virttest/env_process.py

```diff
@@ -928,7 +928,10 @@ def preprocess(test, params, env):
     # done as root, here we do a check whether
     # we satisfy that condition, if not try to make it off
     # otherwise throw TestError with respective error message
-    cpu_family = cpu_utils.get_family() if hasattr(cpu_utils, 'get_family') else cpu_utils.get_cpu_arch()
+    try:
+        cpu_family = cpu_utils.get_family() if hasattr(cpu_utils, 'get_family') else cpu_utils.get_cpu_arch()
```

Comment:
What sounds strange to me here is that cpu_family is getting a "cpu_arch()". So basically you are trying to get the family but if dont have this method, then getting the arch. I'm assuming that this was some change inside the library, but even with that, sounds strange to have a `cpu_family = some arch`.

Reply:
> What sounds strange to me here is that cpu_family is getting a "cpu_arch()". So basically you are trying to get the family but if dont have this method, then getting the arch. I'm assuming that this was some change inside the library, but even with that, sounds strange to have a `cpu_family = some arch`.

I'd have to look into the CPU changes again, but isn't just the terminology that changed, and the result from both are expected to be the same?

Reply:
@beraldoleal yep, naming is not great, some history to justify, old cpu utils was not having right naming for methods, that was the whole reason behind  cpu utils rework as part of https://github.com/avocado-framework/avocado/pull/3350

this work around is make this work with older avocado(69lts) working with latest avocado-VT.

Reply:
basically we were getting power9, power8 as output for get_cpu_arch() in older avocado cpu utils which was not correct, it was supposed to be powerpc, and power9 and power8 are cpu families, so new cpu utils changed them, this handle is just to
workaround for 69lts, so we are good.